Source/Core/DolphinQt/NetPlay/NetPlaySetupDialog.cpp

```diff
@@ -110,6 +118,12 @@ void NetPlaySetupDialog::CreateMainLayout()
 
   m_host_port_box->setMaximum(65535);
   m_host_force_port_box->setMaximum(65535);
+  m_host_chunked_upload_limit_box->setRange(1, 1000000);
+  m_host_chunked_upload_limit_box->setSingleStep(100);
+  m_host_chunked_upload_limit_box->setSuffix(QStringLiteral(" kbps"));
```

Comment:
Is it bits per second or bytes per second?

Reply:
Traditionally, "b" means bits and "B" means bytes.

Reply:
I got confused by the "bytes_per_second" in the implementation, but you do a /8 over there that I missed :) carry on.

Reply:
Yeah, the implementation converts to bytes because at this abstraction level we use bytes, not bits for moving data around. I use bits for the config though because network speed is pretty much always measured in bits per second.